.concourse/pipeline.yml

```diff
@@ -75,3 +97,60 @@ jobs:
         - deploy
         - --settings
         - .concourse/settings.xml
+
+- name: build-pr
+  public: true
+  serial: true
+  plan:
+    - get: pull-request
+      params:
+        integration_tool: checkout
+        list_changed_files: true
+        submodules: true
+      trigger: true
+      version: every
+    - in_parallel:
+      - file: pull-request/.git/resource/head_sha
+        format: trim
+        load_var: commit
+        reveal: true
+      - file: pull-request/.git/resource/pr
+        format: trim
+        load_var: pull_request_key
+        reveal: true
+      - get: keycloak
+        trigger: true
+        params:
+          globs:
+            - do-not-download-artifacts
+
+    - load_var: keycloak_version
+      file: keycloak/version
+      format: trim
+      reveal: true
```

Comment:
The sequencing here is wonky. You should fetch the keycloak release in parallel with the PR. You can make the `load_var` operations sequential with their respective sources.